### ISSUE TITLE
Empty sfGuardUser

### ISSUE THREAD

**Issue by tobiassjosten** [fabriceb/sfFacebookConnectPlugin#9]
My newly created sfGuardUser doesn't have any information in it, other than the Facebook ID. Is perhaps because of a change on the FB side?

When I try the Graph API to poll information about my user, it's all there. As documented on http://developers.facebook.com/docs/reference/api/user:

$fb_uid = sfFacebook::getAnyFacebookUid();
$ret = sfFacebook::getFacebookApi($fb_uid);
var_dump($ret);

**Comment by tristanbes** [comment]
I just ran a quick test, and i have the same problem.
sfGuardUser is empty appart facebook ID.
sfGuardUserProfile is empty too (email / email_hash)

Is anyybody found somthing ?

**Comment by jbcayrou** [comment]
Yes, i have the same problem, with an error when i tried to do sfFacebook::getFacebookClient->api('/me');
I try to find a solution. I noticed if I signin as that example https://github.com/facebook/php-sdk/blob/master/examples/example.php It works.
So i think it may come from connexion script
